public/src/js/widgets/columns/fronts-config.html

```diff
@@ -126,7 +126,10 @@
              <!-- ko if: id -->
                 <div class="instructions">
                     <!-- ko if: !state.isOpenProps() && collections.items().length -->
-                        <span class="linky tool--metadata" data-bind="click: openProps">edit metadata</span>
+                        <span class="linky tool--metadata" data-bind="
+                          click: openProps,
+                          visible: !state.isMetaLocked
```

Comment:
does this works for other priorities? you only define `state.isMetaLocked` when priority is `advertisement`. Which by the way should be an observable, just for consistency